pebblo/app/daemon.py

```diff
@@ -1,22 +1,32 @@
-import uvicorn
-from fastapi import FastAPI
-from pebblo.app.routers.routers import router_instance
-
 from pebblo.topic_classifier.topic_classifier import TopicClassifier
 from pebblo.entity_classifier.entity_classifier import EntityClassifier
 
+from pebblo.app.config.config import load_config
+import sys
+
+from pebblo.app.config.service import Service
+
+config_details = {}
+
 
 def start():
     # Init TopicClassifier(This step downloads the models and put in cache)
     _ = TopicClassifier()
     # Init EntityClassifier(This step downloads all necessary training models)
     _ = EntityClassifier
 
-    # Initialise app instance
-    app = FastAPI()
+    # CLI input details
+    cli_input = sys.argv[1:]
+    cli_str = ' '.join(cli_input)
+    global config_details
 
-    # Register the router instance with the main app
-    app.include_router(router_instance.router)
+    # For loading config file details
+    if '--help --config' in cli_str:
```

Comment:
Why do we need `--help` here? We can have separate handling for `--help`

Reply:
updated the command
resolving.